services/static-webserver/client/scripts/post-compile.py

```diff
@@ -0,0 +1,61 @@
+import os
+import json
+
+
+output_folders = [
+    "source-output", # dev output
+    "build-output",  # default production output
+    "build-client"   # I believe we create the production outputs here
+]
+
+
+def read_json_file(filename):
+    dirname = os.path.dirname(__file__)
+    meta_filename = os.path.join(dirname, filename)
+    f = open(meta_filename)
+    metadata = json.load(f)
+    return metadata["applications"]
+
+
+def update_apps_metadata():
+    dirname = os.path.dirname(__file__)
```

Comment:
This can be a constant in the module. You are also using it in multiple places. 
This is pretty common in our code. We do it this way

https://github.com/itisfoundation/osparc-simcore/blob/cf422bb2b61b8ae738eaad79f34f1de7c2bce995/packages/pytest-simcore/tests/test_fixture_environs.py#L4